sdk/clientcore/core/src/main/java/io/clientcore/core/credentials/TokenCredential.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.core.credentials;
+
+/**
+ * <p>
+ * Token Credential interface serves as a fundamental component for managing and providing access tokens.
```

Comment:
```suggestion
 * The TokenCredential interface serves as a fundamental component for managing and providing access tokens.
```

Reply:
`{@link TokenCredential}`? (I'm never sure how many links is too many links but I feel like "always link typenames" is a good rule?)